### PR TITLE
e2e for 4-auth

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

**Comment by bshaffer** [comment]
@jeffmendoza @remi PTAL

**Comment by bshaffer** [comment]
jenkins is passing :D

**Comment by jeffmendoza** [comment]
Nice! LGTM

File: 4-auth/spec/features/user_login_e2e_spec.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "spec_helper"
+
+feature "User login (e2e)" do
+
+  scenario "Logging in (e2e)", :e2e => true do
+    visit E2E.url + root_path
+    expect(page).not_to have_content "Example User"
```
Comment (remi) [inline 4-auth/spec/features/user_login_e2e_spec.rb]: Remove

File: 4-auth/spec/features/user_login_e2e_spec.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    expect(page).not_to have_link "logout"
```
Comment (remi) [inline 4-auth/spec/features/user_login_e2e_spec.rb]: Remove

These assertions never flip to true

**Comment by remi** [comment]
LGTM

**Pull request merged**